src/plugin.ts

```diff
@@ -92,7 +92,7 @@ export const isBooleanValueInvariant = (ast: ComparisonNode): void => {
  * > was to have a solution for following `pg` problem: [parameterized query with an
  * > `IN` operator](https://github.com/brianc/node-postgres/issues/1452).
  */
-export const MapInToEqualsAnyPlugin: RsqlOperatorPlugin = {
+export const MapInToEqualsAnyPlugin = {
```

Comment:
best to have this use `satisfies` to avoid having the `invariant` func reported as optional